kafka-connect-aws-s3/src/main/scala/io/lenses/streamreactor/connect/aws/s3/source/reader/ReaderManager.scala

```diff
@@ -58,8 +59,8 @@ class ReaderManager(
 
     def readNextFile: ReaderState =
       fileSource.next() match {
-        case Left(exception: Throwable) =>
-          toExceptionState(exception)
+        case Left(exception: FileListError) =>
```

Comment:
Throwable avoid the match not being exhaustive. Just checking, there can not be any other exception but FileListError, otherwise the match will raise an exception

Reply:
Yes, it is. just saw the code.